pkgs/top-level/all-packages.nix

```
@@ -13514,16 +13512,16 @@ with pkgs;
 
   any-nix-shell = callPackage ../shells/any-nix-shell { };
 
-  bash = lowPrio (callPackage ../shells/bash/5.1.nix {
+  bash = lowPrio (callPackage ../shells/bash/5.2.nix {
```

```suggestion
  bash = lowPrio (callPackage ../shells/bash/5.nix {
```
We might as well remove it to make updates easier